test/com/google/enterprise/adaptor/TestHelper.java

```diff
@@ -87,17 +89,6 @@ public void setAuthzAuthority(AuthzAuthority authzAuthority) {}
     return getDocIds(adaptor, Collections.<String, String>emptyMap());
   }
 
-  public static byte[] getDocContent(Adaptor adaptor, DocId docId)
```

Comment:
None of our connectors use the test classes from the library. They all reference the library through adaptor-withlib.jar. Four connectors have copies of TestHelper, but none of them use this one, or anything out of WrapperAdaptor except for WrapperAdaptorContext as noted. If third-party connectors are referring to things in the library tests, I have no sympathy.